docs/changelog.rst

```diff
@@ -1,22 +1,30 @@
 Changelog
 ================================
-CBC SDK 1.3.0 - Released ?, 2021
+CBC SDK 1.3.0 - Released June 8, 2021
 --------------------------------
 
-Bug Fixes
-
-  * Fix in the sort_by for Query class
-  * Fix the ability to set expiration for binary download URL
-  * Fix bug in helpers read_iocs functionality
-
 New Features
 
-* Added User Management, Grants, Access Profiles, Permitted Roles
-* Refactor of the vulnerability models
+* Add User Management, Grants, Access Profiles, Permitted Roles
+* Mov Vulnerability models to Platform package in preparation for supporting Endpoints and Workloads
+* Refactor Vulnerability models
+  * ``VulnerabilitySummary.get_org_vulnerability_summary`` static function changed to ``Vulnerability.OrgSummary`` model with query class
```

Comment:
Formatting didn't work - think it needs a newline between the first and then second level bullet